---
bool result
string[] label
float32[] confidence
float32[] x
float32[] y
float32[] z
float32[] width
float32[] height
float32[] depth
float32[] min_x
float32[] max_x
float32[] min_y
float32[] max_y
float32[] min_z
float32[] max_z